src/pages/registration/resource_type_tab/components/PublicationChannelOption.tsx

```diff
@@ -9,15 +9,23 @@ interface PublicationChannelOptionProps {
   props: HTMLAttributes<HTMLLIElement>;
   option: Journal | Series | Publisher;
   state: AutocompleteRenderOptionState;
+  hideScientificLevel?: boolean;
 }
 
-export const PublicationChannelOption = ({ props, option, state }: PublicationChannelOptionProps) => (
+export const PublicationChannelOption = ({
+  props,
+  option,
+  state,
+  hideScientificLevel = false,
```

Comment:
`Undefined` er falsy, om det ikke var meningen å være eksplisitt da

```suggestion
  hideScientificLevel,
```

Reply:
tanken var å være eksplisitt ja, syns det gir mening for props at de blir `true | false` i stedet for `true | false | undefined`, selv om det ikke burde påvirke noe i praksis